### DISCUSSION TITLE
Udah ditambah API dari source rekomen tpi dia ngga bisa login lgi sbagai admin

### DISCUSSION THREAD

**Discussion by TachooDa** [rizkyilhampra/spdhtc#77] [Q&A] [answered]
sebelumnya itu udah bro ngisi api nya dari source rekomen dan bisa, tpi entah kenapa sekarang ngga bisa lgi dan coba ambil API baru lagi kan
error ini muncul lagi, padahal sebelumnya udah bisa
![image](https://github.com/user-attachments/assets/4e14e654-b8e8-4a1a-b13d-774c54ccb92f)

**Comment by rizkyilhampra** [comment, marked as answer]
1. Pastikan *variable* `RAJAONGKIR_API_KEY` di file `.env` telah diisi dengan benar
2. Jalankan `php artisan optimize:clear`
3. Hapus semua file yang ada di folder `storage/framework/sessions` (Logout)
4. Coba kembali
5. Jika tetap sama, jalankan kembali `php artisan migrate:fresh --seed`

**Reply by TachooDa** [reply]
oke bro, udah bisa
btw bro perintah kaya php artisan itu ada di doc nya laravel ya?

**Reply by TachooDa** [reply]
mao nanya lagi bro kalo di next first gejala itu ngga muncul dan masih not found, padahal udah di atur di code nya apa harus migrate dulu buat db nya?

**Reply by TachooDa** [reply]
udah solve bro 
sorry ngerepotin
Terima Kasih
